### ISSUE TITLE
News demo fails WCAG 2.1 -- Focus visible

### ISSUE THREAD

**Issue by bsouster** [wet-boew/GCWeb#1644]
I count 5 invisible tabs between the filter UI and the first entry.

![tab-invis](https://user-images.githubusercontent.com/1944744/75639828-a539b780-5c00-11ea-8ebe-4dc1c990e0a2.jpg)

Success criteria:
* [Focus visible](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible)

**Comment by EricDunsworth** [comment]
It seems to be the result of the ``wb-inv`` class being set on the filterable table's ``thead`` element. The 5 invisible tabs are focusing onto 5 visually-hidden interactive ``th`` elements. Each one sorts their respective column's rows in ascending/descending order.

IMO the DataTables plugin is being misused by both WET 4.0 and GCWeb to present filterable data a completely different way than how that plugin was intended to be used. The ``nws-tbl-*`` classes that are used to present the table cells as "normal" content also don't seem to be documented anywhere.

Going back to this specific issue...
* Modifying the plugin to set a ``hidden`` attribute on the ``thead`` element (instead of ``wb-inv``) would fix the hidden focus issue. But that would likely also result in screen readers treating the table as a layout table. That might not be ideal since if the data were to be presented like that, it wouldn't be in a very structured way.
* Another possible way would be to make the ``th`` elements non-interactive when the data isn't visually-presented as tabular data... although it'd still be pretty weird. Presenting the data as a table to screen readers and as plain content to sighted users...

**Comment by bsouster** [comment]
Can we just inject tabindex=-1 on the invisible elements?

**Comment by bsouster** [comment]
Yeah, see, here's the problem. Tabindex="0"
![tabindex](https://user-images.githubusercontent.com/1944744/76086808-d9a3cf80-5f82-11ea-8663-5ff856281ddc.jpg)

**Comment by EricDunsworth** [comment]
@duboisp @GormFrank What are your takes on this?

**Comment by GormFrank** [comment]
One thing for sure is that just changing the tabindex=0 to tabindex=-1 would not be sufficient in my opinion since the TH elements have an aria-control attribute on them. It is a tricky one. @duboisp
